perfmetrics/scripts/gsheet/gsheet.py

```diff
@@ -12,6 +12,7 @@ def _get_sheets_service_client():
   service = build('sheets', 'v4', credentials=creds)
   return service
 
+
 def write_to_google_sheet(worksheet: str, data) -> None:
   """Calls the API to update the values of a sheet.
```

Comment:
Fix these comments here and other functions too

Reply:
done